en/docs/design/api-security/api-authentication/securing-apis-using-certificate-bound-access-tokens.md

```diff
@@ -0,0 +1,124 @@
+
+# Securing APIs with Certificate Bound Access Tokens
+
+While OAuth 2.0 relies on bearer tokens for securing access to protected resources, bearer tokens require Transport Layer Security between an OAuth client and the resource server when presenting the access token to get access to a resource. In this security model, it is crucial to prevent access token leakage. Therefore, access token storage and transfer have to be done with care.
+
+APIs in WSO2 API Manager can be secured using Certificate Bound Access Tokens, also known as Holder of Key Tokens. The following section explains as to how the APIs in WSO2 API Manager can be secured using Certificate Bound Access Tokens.
+
+## Import client certificate to the truststore
+
+ Follow the steps given below to import the CA-signed public key certificate into API Manager's default truststore (client-truststore.jks).
+ 
+1.  Get a copy of the `client-truststore.jks` file from the `<API-M_HOME>/repository/resources/security/` directory.
+2.  Export the public key from your .jks file using the following command.
+
+    ``` bash
+    keytool -export -alias certalias -keystore newkeystore.jks -file <public key name>.pem
+    ```
+
+3.  Import the public key you extracted in the previous step to the `client-truststore.jks` file using the following command.
+
+    ``` bash
+    keytool -import -alias certalias -file <public key name>.pem -keystore client-truststore.jks -storepass wso2carbon
+    ```
+
+    !!! Note
+    
+        `wso2carbon` is the keystore password of the default `client-truststore.jks` file.
+
+## Enable the Certificate Bound Access Token
+
+1. Open the `<API-M_HOME>/repository/conf/deployment.toml` file.
+
+2. Configure the `enable_outbound_auth_header` under the `[apim.oauth_config]` configuration.
+
+    ``` tab="Format"
+    [apim.oauth_config]
+    enable_certificate_bound_access_token = true
+    ```
+
+    ``` tab="Example"
```

Comment:
Is it correct that the format and the example have the same content?

If there is no difference between the two, we do not need to repeat the content.

Reply:
fixed with a6b62f9bd1d6e2965c1980c2bfe2ff7c81bfdd5b